client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -79,15 +78,12 @@ const VerticalSelect: FunctionComponent< InjectedStepProps > = ( {
 		}
 	};
 
-	const handleSelect = ( vertical: SiteVertical ) => {
-		setSiteVertical( vertical );
-		hideSuggestions();
+	const handleSelect = ( vertical?: SiteVertical ) => {
+		vertical ? setSiteVertical( vertical ) : dirty && resetSiteVertical();
```

Comment:
I'd rather avoid ternaries where we don't really evaluate the expressions on the `?` and `:` branches but use them to call functions. I think a plain ol' `if`/`else` would be fine here.